### DISCUSSION TITLE
How to see data of an activated emergency access ?

### DISCUSSION THREAD

**Discussion by LoicFr** [dani-garcia/vaultwarden#2060] [Q&A]
Hello,

Just to try, I have configured an emergency access (with view access rights) and it is well approved but when I click on the View button of the emergency contact I can't see anything (see picture below). Is that normal ? I am redirecting to https://my.vaultwarden.server.com/#/settings/emergency-access/54a8de73-1da6-4578-b46f-358868d8cba4. What am I supposed to do then ?

![image](https://user-images.githubusercontent.com/38719983/138530818-774b82ee-d1f8-4660-9c8e-c53397b9a76a.png)

Thanks for your help,
Regards,
Loïc

**Comment by BlackDex** [comment]
That means there personal vault is empty. It could be that they only have org shared items. Those you can't see using emergency access.

**Comment by LoicFr** [comment]
Thanks for your answer but this is not the case, the personal vault is not empty. There are also org shared items, but not only.

**Comment by K1ngLear** [comment]
I encountered the same issue. I made a test account to check the emergency access before I grant someone of my family one and I couldn't see the vault of my account, too.

**Reply by BlackDex** [reply]
Strange, it seems to work for me.

**Reply by BlackDex** [reply]
If you first go to the 'My Vault' page, then press F12 to open the Browsers Developer Console
And after that go to 'Settings' > 'Emergency Access', ask to view the vault of that person, do you see any errors?
There should be network call to `https://vw.domain.tld/api/emergency-access/[UUID]/view` which should have a response with json and the ciphers of that user.

**Reply by LoicFr** [reply]
I have this error when following your instructions:

core.js:6210 ERROR Error: Uncaught (in promise): TypeError: Cannot read properties of null (reading 'split')
TypeError: Cannot read properties of null (reading 'split')
    at e.<anonymous> (crypto.service.ts:556)
    at main.5c0db323b598138d1784.js:1
    at Object.next (main.5c0db323b598138d1784.js:1)
    at main.5c0db323b598138d1784.js:1
    at new t (zone.js:1340)
    at o (main.5c0db323b598138d1784.js:1)
    at e.rsaDecrypt (main.5c0db323b598138d1784.js:1)
    at e.<anonymous> (emergency-access-view.component.ts:75)
    at main.5c0db323b598138d1784.js:1
    at Object.next (main.5c0db323b598138d1784.js:1)
    at P (zone.js:1255)
    at zone.js:1162
    at s (main.5c0db323b598138d1784.js:1)
    at t.invoke (zone.js:400)
    at Object.onInvoke (core.js:28591)
    at t.invoke (zone.js:399)
    at e.run (zone.js:160)
    at zone.js:1318
    at t.invokeTask (zone.js:434)
    at Object.onInvokeTask (core.js:28578)

**Reply by BlackDex** [reply]
Hmm something within the response json is breaking the web-vault.

**Reply by antipasta** [reply]
I'm having this issue as well. I tested both with manually approving access, and letting the number of days lapse.

I don't have the console output handy but I do recall it being about 'split', so I think it's the same as what LoicFr pasted. If it matters, my vault was originally imported from keepass.

**Reply by BlackDex** [reply]
To see what is breaking this we need a json response which breaks. Since i can't reproduce this currently, and i think neither do the other main contributors. I understand that people are a bit hesitant to provide this, but all is encrypted, so we can't extract anything.

I also think the it is somewhere in the `Ciphers` json tree, and that if there is a `KeyEncrypted` and `Object` key/value within the response that that is probably not the issue, and we do not need that.

Please contact me on Matrix  https://matrix.to/#/#vaultwarden:matrix.org, I am `@blackdex:matrix.org`
